src/test/integration/org/apache/cassandra/sidecar/routes/tokenrange/BaseTokenRangeIntegrationTest.java

```diff
@@ -291,4 +294,17 @@ void assertMappingResponseOK(TokenRangeReplicasResponse mappingResponse,
             .isGreaterThanOrEqualTo(replicationFactor);
         }
     }
+
+    private void validateRanges(List<TokenRangeReplicasResponse.ReplicaInfo> replicaRanges)
+    {
+        // Ranges should not be empty
+        replicaRanges.stream().forEach(r -> assertThat(r.start()).isNotEqualTo(r.end()));
+        // Ranges should include partitioner start and end
+        replicaRanges.stream()
+                     .map(TokenRangeReplicasResponse.ReplicaInfo::start)
+                     .anyMatch(s -> s.equals(Murmur3Partitioner.MINIMUM.toString()));
+        replicaRanges.stream()
+                     .map(TokenRangeReplicasResponse.ReplicaInfo::end)
+                     .anyMatch(s -> s.equals(Long.toString(Murmur3Partitioner.MAXIMUM)));
```

Comment:
should we assert something here?
```suggestion
        assertThat(replicaRanges.stream()
                                .map(TokenRangeReplicasResponse.ReplicaInfo::start)
                                .anyMatch(s -> s.equals(Murmur3Partitioner.MINIMUM.toString()))).isTrue();
        assertThat(replicaRanges.stream()
                                .map(TokenRangeReplicasResponse.ReplicaInfo::end)
                                .anyMatch(s -> s.equals(Long.toString(Murmur3Partitioner.MAXIMUM)))).isTrue();
```

Reply:
ah, yes, needs an assert, thanks.